src/framework/parsers/glb-parser.js

```diff
@@ -721,7 +721,9 @@ const createDracoMesh = (device, primitive, accessors, bufferViews, meshVariants
             } else {
                 // create vertex buffer
                 const numVertices = decompressedData.vertices.byteLength / vertexFormat.size;
-                Debug.assert(numVertices === accessors[primitive.attributes.POSITION].count, 'mesh has invalid draco sizes');
+                if (numVertices !== accessors[primitive.attributes.POSITION].count) {
+                    Debug.warn('mesh has invalid draco sizes');
+                }
```

Comment:
add the whole thing inside `Debug.call(() => { }` so that even the condition gets stripped out
and maybe add that glb name if possible
and warnOnce could be good as well

Reply:
+1 on some identifier would be great 
![image](https://github.com/playcanvas/engine/assets/16639049/da4af268-3085-4bc4-8a53-6083c9a06652)

Reply:
Projects don't need reimport TBH. We'll patch the engine with this change now so we can get it out quickly, then add more debugging information later.